actors/abi/deal.go

```diff
@@ -1,10 +1,10 @@
 package abi
 
-import "math/big"
+import big "github.com/filecoin-project/specs-actors/actors/abi/bigint"
 
 type DealID int64
 type DealIDs struct {
 	Items []DealID
 }
 
-type DealWeight *big.Int
+type DealWeight = big.Int
```

Comment:
Let's document this decision.
```
BigInt types are aliases rather than new types because the latter introduce incredible amounts of noise converting to and from types in order to manipulate values. We give up some type safety for ergonomics.
```